tempesta_fw/mbedtls/lk.c

```diff
@@ -0,0 +1,18 @@
+#include "lk.h"
+
+void *calloc(size_t n, size_t size)
+{
+	return kzalloc(n * size, GFP_ATOMIC);
```

Comment:
This dummy functiona redefinition can work for very first version of code. We have fast pool allocation which should be used instead of poor `kmalloc()`. Moreover, we need carefully review all allocations and do our best to avoid them.